Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/UserDataActivity.kt

```diff
@@ -73,18 +80,34 @@ class UserDataActivity : AppCompatActivity() {
 
         mBinding.buttonExportUserData.setOnClickListener { exportUserData() }
 
+        mBinding.buttonResetSettings?.setOnClickListener { confirmResetSettings() }
```

Comment:
Looks like you haven't added the button to `layout-land/activity_user_data.xml`. If you do that, the `?` here will be unnecessary.